sasdata/data_util/manipulations.py

```diff
@@ -63,19 +63,18 @@ def get_q_compo(dx: float, dy: float, detector_distance: float, wavelength: floa
     return out
 
 
-def flip_phi(phi: float) -> float:
+def normalize_angle(phi: float, closed_at_end: bool = False) -> float:
```

Comment:
I agree that the old nomenclature makes no sense.  I will admit to being too chicken myself to change it however. That is a brave move.   Are you absolutely sure than this function is not called from any other modules in the sasview code base?

Reply:
I've double checked, and yes `flip_phi()` was only ever called within `manipulations.py`, and never in sasview. So this rename causes no problems.